code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -178,28 +180,26 @@
 				return
 			boxes += add
 			newbox.boxes.Cut()
-			to_chat(user, span_notice("You put [newbox] on top of [src]!"))
 			newbox.update_appearance()
 			update_appearance()
 			user.regenerate_icons()
-			if(boxes.len >= 5)
-				if(prob(10 * boxes.len))
-					to_chat(user, span_danger("You can't keep holding the stack!"))
+			if(length(boxes) >= 5)
+				if(prob(10 * length(boxes)))
+					balloon_alert(user, "dropped it!")
```

Comment:
```suggestion
					balloon_alert(user, "oops!")
```

nit, i just think this reads weird

Reply:
i had no idea what to put in this one and "oops" was the simplest and best option the whole time 😔